nav2_bt_navigator/behavior_trees/navigate_w_replanning_and_recovery.xml

```diff
@@ -18,12 +18,15 @@
           <ClearEntireCostmap name="ClearLocalCostmap-Context" service_name="local_costmap/clear_entirely_local_costmap"/>
         </RecoveryNode>
       </PipelineSequence>
-      <SequenceStar name="RecoveryActions">
-        <ClearEntireCostmap name="ClearLocalCostmap-Subtree" service_name="local_costmap/clear_entirely_local_costmap"/>
-        <ClearEntireCostmap name="ClearGlobalCostmap-Subtree" service_name="global_costmap/clear_entirely_global_costmap"/>
-        <Spin spin_dist="1.57"/>
-        <Wait wait_duration="5"/>
-      </SequenceStar>
+      <ReactiveFallback name="RecoveryFallback">
```

Comment:
Please show images of the new BTs for any BTs you change with the tool. You'll also need to update those images in the readme files / anywhere else the BT images are displayed. Also look over the readmes for descriptions of these BTs and update them with this new logic flow.

Reply:
Okay, I'll include the documentation changes in the new pr.